web/src/app/new-category/page.tsx

```diff
@@ -1,16 +1,39 @@
 import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
 import Link from "next/link";
+import CompanyLogo from "../components/company-logo";
 
 export default function NewCategory() {
     return (
-        <section className="flex flex-col justify-center items-center h-screen">
-            <div className="flex flex-col items-center space-y-2">
-                <h1 className="text-xl font-bold">
-                    Tela para adicionar categorias
-                </h1>
-                <Button>
-                    <Link href="/">Voltar para home</Link>
-                </Button>
+        <section>
+            <div className="h-40 px-6 py-12">
+                <CompanyLogo />
+            </div>
+            <div className="flex flex-col p-6 gap-12">
+                <div>
+                    <h2 className="text-3xl font-semibold text-indigo-950">
+                        Nova Categoria
+                    </h2>
+                </div>
+                <form className="flex flex-col gap-6">
+                    <div className="flex flex-col gap-4">
+                        <Label htmlFor="NameCategory">Nome da categoria</Label>
+                        <Input
+                            type="nome-categoria"
+                            placeholder="Digite o nome da categoria"
+                        />
+                    </div>
+                    <div className="flex flex-col gap-4">
+                        <Button className="bg-indigo-950 text-white">
+                            SALVAR
+                        </Button>
+
+                        <Button className="bg-white text-indigo-950 border border-indigo-950">
```

Comment:
Mesma coisa, adicionar font-bold, para padronizar todos os botões do app

Reply:
Esse Button aqui vai submeter o form se o usuário clicar nele. O que vc que é que se pareça com um botão mas funcione como link. Pra isso, vc vai precisar adicionar a prop `asChild` aqui.

```suggestion
                        <Button asChild className="bg-white text-indigo-950 border border-indigo-950">
```